pkg/chunk/cassandra/sessions.go

```diff
@@ -0,0 +1,42 @@
+package cassandra
+
+import (
+	"sync"
+
+	"github.com/gocql/gocql"
+)
+
+var (
+	sharedSessions map[string]*sharedSession
+	sessionsMtx    sync.Mutex
```

Comment:
These are still globals.  If we give easy session its own name, we won't get registrations error, and we won't need globals.

Reply:
Sorry, I saw your review after pushing the code. I thought we didn't want to create multiple sessions per schema config where Cassandra is use so I went this way which I accept is ugly.
I didn't get a chance to work on this yet. Will fix this.